CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.34.8
```

Comment:
Change log entries should exist in a pending log entry and not modify the actual CHANGELOG file 👍

Reply:
What's the correct way of creating .pending entry now?

Reply:
`go get github.com/cosmos/tools/cmd/clog`

Then:

```
clog add -t keys improvements
```